common/k8stest/util_rdma.go

```diff
@@ -132,3 +132,107 @@ func IsVolumeAccessibleOverTcp(volUuid string) (bool, error) {
 	}
 	return false, nil
 }
+
+func RemoveRdmaDeviceOnNode(node string) error {
+	rdmaDeviceList, err := ListRdmaDevice(node)
+	if err != nil {
+		return err
+	}
+	if len(rdmaDeviceList) != 0 {
+		logf.Log.Info("RDMA device found", "node", node, "list", rdmaDeviceList)
+		//create rdma device
+		nodeIp, err := GetNodeIPAddress(node)
+		if err != nil {
+			return fmt.Errorf("failed to get node %s ip, error: %v", node, err)
+		}
+		var iface, out string
+		for _, device := range rdmaDeviceList {
+			if device.IfName == RdmaDeviceName {
+				// get interface name
+				iface = e2e_config.GetConfig().NetworkInterface
+				out, err = e2e_agent.DeleteRdmaDevice(*nodeIp, RdmaDeviceName)
+				if err != nil {
+					return err
+				}
+			}
+		}
+		if iface == "" {
+			return fmt.Errorf("rdma device %s not found", RdmaDeviceName)
+		}
+
+		logf.Log.Info("Device deleted", "node", node, "output", out, "interface", iface, "device", RdmaDeviceName)
+
+	}
+	return nil
+}
+
+func DisableRdmaOnNode(node string) error {
+	//disable rdma on the io-engine node
+	platformName := e2e_config.GetConfig().Platform.Name
+	if platformName == "Maas" {
```

Comment:
Strictly speaking this isn't true, it isn't the platform but rather whether the node has RDMA capable hardware.
Is there a way to delegate this decision re: `disable network interface` or `remove rdma device`  to the e2e-agent executing on the node?
IMO that would be the ideal solution, for e.g. it would not matter if  all nodes in a cluster have RDMA capable hardware or not.

Reply:
ideal solution will be like that only but there is no easy way to find out Hardware capable RDMA node .
And by going with current approach, we have visibility what's happening on node, whether interface is brought down or rdma device is disabled with respect to platform otherwise you have to check e2e-agent log and verify it.

Reply:
For now please add a FIXME statement

Reply:
Added fixme for the time being https://github.com/openebs/openebs-e2e/compare/9417197bb5a99395ba485cc8eb0fc62c21361aaf..f63b0b6a28b18137b5a82c41dbc8320dc06e31ea